pkgs/development/libraries/raylib/default.nix

```
@@ -0,0 +1,30 @@
+with import <nixpkgs> { };
```

@svantepolk Could you please use:

```suggestion
{ stdenv
, fetchGitHub
, libGl
, xorg
, xlibs
, cmake
}:
```

Also, note the usage of `fetchFromGitHub`.